subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -812,7 +812,15 @@ static void isr_done(void *param)
 	/* Get reference to CIS LLL context */
 	cis_lll = param;
 
-	/* Generate ISO Data Invalid Status */
+	/* Adjust sn when flushing Tx */
+	bn = bn_tx;
+	while (bn <= cis_lll->tx.bn) {
+		cis_lll->sn++;
+
+		bn++;
+	}
```

Comment:
This basically adds the difference between `bn` and `cis_lll->tx.bn`, right? 

Would it be simpler to do
```c
cis_lll->sn += cis_lll->tx.bn - bn
```

Reply:
Yes. I wanted to keep it simple as current usecase where BN=1. Will do some testing and update the PR.